src/unlabel.ml

```diff
@@ -0,0 +1,192 @@
+let slurp_file file =
+  let ch = open_in file in
+  let buf = Buffer.create (min 1024 (in_channel_length ch)) in
+  try
+    while true do Buffer.add_channel buf ch 4096 done;
```

Comment:
this might be buggy on old versions of OCaml, I remember propositing a patch for it… The last chunk, if smaller than 4096, might be dropped. To be checked…

Reply:
works for me on OCaml 4.03.0. Won't be used on 4.02.*